scout/server/blueprints/clinvar/templates/multistep_add_variant.html

```diff
@@ -85,24 +85,34 @@ <h2 class="fs-title">Variant Details</h2>
                     <span class="text-danger" data-bs-toggle='tooltip' title="If you do not provide any HGVS expression, chromosome coordinates will be used to describe this variant instead (automatic). HGVS expressions were validated using VariantValidator"><strong>?</strong></span>
                     <span class="badge bg-primary float-end"><a class="text-white" href="https://variantvalidator.org/service/validate/" target="_blank" rel="noopener">VariantValidator</a></span>
                     <br><br>
+                    <table style="width:100%; table-layout: auto;">
```

Comment:
Hm, not the biggest fan: but I'm sure it will work - they did in like 1995. 😉

Reply:
Oh, we are supposed to add some aria style suggestion to tell it is just a layout table, not a content one, aren't we?